src/main/java/org/retest/rebazer/connector/BitbucketConnector.java

```diff
@@ -124,10 +121,11 @@ private DocumentContext jsonPathForPath( final String urlPath ) {
 
 	@Override
 	public void addComment( final PullRequest pullRequest, final String message ) {
-		final Map<String, String> request = new HashMap<>();
-		request.put( "content", message );
+		final Map<String, BitbucketContent> request = new HashMap<>();
+		final BitbucketContent comment = BitbucketContent.builder().raw( message ).build();
+		request.put( "content", comment );
```

Comment:
You could also include the map in `BitbucketComment`, something like:

```java
@Value
public class BitbucketComment {
    private static final String KEY = "content";
    String raw;
    Map<String, String> request() {
        final Map<String, String> request = new HashMap();
        request.put( KEY, raw );
        return request;
    }
}
```

Then you could do:

```java
BitbucketComment comment = new BitbucketComment( message );
template.postForObject( requestPath( pullRequest ) + "/comments", comment.request(), String.class );
```

However, @modulo11 is more familiar with Spring and its best practices.

Reply:
I spoke to @modulo11 a little bit about it and we decided to solve this problem with a simpler solution without the Lombok library